src/public/Interfaces.ts

```diff
@@ -155,6 +155,19 @@ export interface ExecutionContext {
     functionDirectory: string;
 }
 
+export interface TraceContext {
+    /** Populated during function invocation and corresponds to Activity.Current.Id during the invocation. */
```

Comment:
Can we make this a more user-facing comment? Ex: 
`An ID that describes where in a tracing system the incoming request is.`